src/NzbDrone.Core/Notifications/Discord/Discord.cs

```diff
@@ -235,6 +235,44 @@ public override void OnRename(Series series)
             _proxy.SendPayload(payload, Settings);
         }
 
+        public override void OnDelete(EpisodeDeleteMessage deleteMessage)
+        {
+            var series = deleteMessage.Series;
+            var episodes = deleteMessage.EpisodeFile.Episodes;
+
+            var attachments = new List<Embed>
+                              {
+                                  new Embed
+                                  {
+                                      Title = GetTitle(series, episodes)
+                                  }
+                              };
+
+            var payload = CreatePayload("Episode Deleted", attachments);
+
+            _proxy.SendPayload(payload, Settings);
+        }
+        public override void OnDelete(SeriesDeleteMessage deleteMessage)
+        {
+            var series = deleteMessage.Series;
+            string deletedFiles = deleteMessage.DeleteFiles ?
+                "Removed from Sonarr and disk" :
+                "Removed from Sonarr but NOT disk";
```

Comment:
`Series removed, files were not deleted`

Reply:
Resolved